pkg/vere/db/lmdb.c

```diff
@@ -38,11 +38,20 @@ intmax_t mdb_get_filesize(mdb_filehandle_t han_u);
 /* u3_lmdb_init(): open lmdb at [pax_c], mmap up to [siz_i].
 */
 MDB_env*
-u3_lmdb_init(const c3_c* pax_c, size_t siz_i)
+u3_lmdb_init(const c3_c* pax_c)
```

Comment:
Lets restore the size argument here, we really should be able to set this parameter at runtime / via commandline flag.